scripts/buidler-hooks.js

```diff
@@ -28,11 +27,9 @@ module.exports = {
   // Called when the start task needs to know the app proxy's init parameters.
   // Must return an array with the proxy's init parameters.
   getInitParams: async ({}, { web3, artifacts }) => {
-    return [
-      42
-    ]
+    return [42];
   },
 
   // Called after the app's proxy is updated with a new implementation.
   postUpdate: async ({ proxy }, { web3, artifacts }) => {}
-}
+};
```

Comment:
Maybe this is prettier, but do we need the semi-colons?

Reply:
Will address in a following PR